.github/workflows/e2e-tests.yml

```
@@ -142,9 +145,13 @@
 
     - name: Run security audit
       run: |
-        npm audit --audit-level=moderate
         echo "## Security Audit Results" >> $GITHUB_STEP_SUMMARY
-        npm audit --audit-level=moderate --json | jq '.vulnerabilities | length' | xargs -I {} echo "Found {} vulnerabilities" >> $GITHUB_STEP_SUMMARY
+        npm audit --audit-level=moderate 2>&1 | tee audit.log || true
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)